config/config.txt

```diff
@@ -615,3 +615,10 @@ ELASTICSEARCH_METRICS_APIKEY thisIsSomethingThatsBased64Encoded==
 
 ## Enable/disable the mentor mrat verb ("Request Mentor Assistance")
 ENABLE_MRAT
+
+## Assets can opt-in to caching their results into `tmp`.
+## This is important, as preferences assets take upwards of 30 seconds (without sleeps) to collect.
+## The cache is assumed to be cleared by TGS recompiling, which deletes `tmp`.
+## This should be disabled (through `CACHE_ASSETS 0`) on development,
+## but enabled on production (the default).
+CACHE_ASSETS 0
```

Comment:
Let's move this into `config/resources.txt` for consistency. Also be sure to update the language so that it clearly states that the option is disabled by default and can be enabled by uncommenting it.